### PR TITLE
Adds Refresh icon

### PR DESCRIPTION
**Pull request by bhanvimenghani**
needs 180 and 181 to be reviewed before reviewing this. solves jira 318
https://www.loom.com/share/3024a6e5812f4d0788d02d5070a3d341?sid=e3266410-58a4-4d31-9825-d5ec8f3a571b

### CODE REVIEW

**Comment by bhanvimenghani** [comment]
image to test the changes of all 3 prs @kusumachalasani  quay.io/bmenghan/pr182

File: src/app/Analytics/LocalMonitoring/RecommendationsForLocalMonitoring/RemoteMonitoring/UsecaseSelection.tsx [hunk 1]
```diff
@@ -43,17 +50,14 @@ const UsecaseSelection = (props: { endTimeArray; setEndTimeArray; SREdata; setSR
...
     const data = await response.json();
     const experimentUsecase = data[0].experiment_usecase_type;
-    if(experimentUsecase) 
-    {const usecase = Object.keys(experimentUsecase).find(
-      key => experimentUsecase[key] === true
-    );
-console.log(usecase)
-    setExpUsecaseType(usecase);
+    if (experimentUsecase) {
+      const usecase = Object.keys(experimentUsecase).find((key) => experimentUsecase[key] === true);
```
Comment (kusumachalasani) [inline src/app/Analytics/LocalMonitoring/RecommendationsForLocalMonitoring/RemoteMonitoring/UsecaseSelection.tsx:57]: Can you consider a case where both local_monitoring and local_experiment can be true ? 

"experiment_usecase_type": {
      "remote_monitoring": false,
      "local_monitoring": true,
      "local_experiment": false
    },

Comment (bhanvimenghani) [reply src/app/Analytics/LocalMonitoring/RecommendationsForLocalMonitoring/RemoteMonitoring/UsecaseSelection.tsx:57]: sure, i will add this change in my current pr. will it be ok?

Comment (bhanvimenghani) [reply src/app/Analytics/LocalMonitoring/RecommendationsForLocalMonitoring/RemoteMonitoring/UsecaseSelection.tsx:57]: I feel it should by default consider as many true values

**Review by kusumachalasani — APPROVED** [review]
LGTM!

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**